Assets/Scripts/GameManager.cs

```diff
@@ -192,8 +198,8 @@ public void LoadGame()
 
         if (PhotonNetwork.IsMasterClient && PressPlayButtonOnce == false)
         {
-            NetworkSceneChangedRaiseEvent();
             MusicChangeRaiseEvent();
```

Comment:
are both Raise event functions necessary?

Reply:
no only the one on line 195 worked